torch_geometric/loader/steps_loader.py

```diff
@@ -0,0 +1,44 @@
+class StepsLoader():
```

Comment:
I personally don't think we need this. This should be automatically supported by adding a custom sampler from PyTorch to `NeighborLoader`, e.g.:
```python
sampler = torch.utils.data.RandomSampler(range(len(train_nodes)), num_samples=num_steps * batch_size)
```

Reply:
This is a very good idea, but it doesn't work with shuffle. So it will work for the inference but not for the training. Of course in training benchmark steps handling can be done just in a train_homo/train_hetero function so if You prefer such option I can switch to the one You proposed.

Reply:
@rusty1s @kgajdamo what about adding it as bench supporting/specific to https://github.com/pyg-team/pytorch_geometric/blob/master/benchmark/utils/utils.py ?

Reply:
Ok, so I checked this RandomSampler and it already do shuffle. So I think we can use it, but then the shuffle argument we pass to NeighborLoader must be set to false. And everything should work.

Reply:
I have made corrections. Please take another look :)